_posts/2024-03-13-java-22.md

```diff
@@ -9,7 +9,7 @@ show_pagenav: true
 
 It was not that long ago that we [first announced support for Java 21]({%
 post_url 2023-11-03-java-21 %}), and subsequently made it a build and runtime
-requirement with [Trino 436]({{site.url}}docs/current/release/release-436.html). 
```

Comment:
Unrelated changes?

Reply:
Sure .. but just removal of whitespace. Not worth a separate commit in my opinion.